SPECS/systemd/systemd.spec

```diff
@@ -178,6 +182,12 @@ install -D -m 0644 %{SOURCE4} %{buildroot}%{_libdir}/systemd/system-preset/99-ma
 %find_lang %{name} ../%{name}.lang
 
 %check
+# Generate machine-id using dbus
+%{_bindir}/dbus-uuidgen --ensure=%{_sysconfdir}/machine-id
+sudo su
```

Comment:
Can we please have a buddy-build with/without check post the rebase.

Reply:
Buddy build:
https://dev.azure.com/mariner-org/mariner/_build/results?buildId=430931&view=results

Reply:
What are you trying to do with this 'sudo su'?

Reply:
Will remove this